plugin/src/main/java/org/wso2/carbon/uuf/maven/bean/AppConfig.java

```diff
@@ -122,6 +125,35 @@ public void setLoginPageUri(String loginPageUri) {
     }
 
     /**
+     * Get session manager implementation class.
```

Comment:
Change javadoc as `Returns session manager class name of this app's config.` 
For getters always try to follow `Returns <something> of this <object type>.` pattern.